src/SafeAnchor.js

```diff
@@ -55,8 +58,18 @@ class SafeAnchor extends React.Component {
     }
   }
 
+  handleKeyDown(event) {
+    event.preventDefault();
+    if (event.key === ' ') {
+      this.handleClick(event);
+    }
+  }
+
   render() {
-    const { componentClass: Component, disabled, ...props } = this.props;
+    const { componentClass: Component, disabled, onKeyDown, ...props } = this.props;
+
+    const handleKeyDown = Component === 'a' ?
+      createChainedFunction(this.handleKeyDown, onKeyDown) : onKeyDown;
```

Comment:
It would be more performant to do this in the constructor, but if the `onKeyDown` prop were changed after the component mounted, the handler wouldn't update. Don't see another way to do this without breaking backwards compatibility.